mod-invoice-storage/examples/voucher.sample

```diff
@@ -0,0 +1,34 @@
+{
+  "id": "a9b99f8a-7100-47f2-9903-6293d44a9905",
+  "systemCurrency": "USD",
+  "invoiceCurrency": "EUR",
+  "exchangeRate": 1.12,
+  "exportToAccounting": false,
+  "voucherType": "Payment",
+  "voucherStatus": "Awaiting Payment",
+  "voucherAmount": 23.45,
+  "voucherDate": "2019-05-06T00:00:00.000+0000",
+  "voucherNumber": "1000",
+  "batchNumber": "101",
+  "invoiceId": "c0d08448-347b-418a-8c2f-5fb50248d67e",
+  "userId": "28d1057c-d137-11e8-a8d5-f2801f1b9fd1",
+  "voucherLines": [
+     {
```

Comment:
Extra spacing. Probably need to be pulled backed by 2 spaces.

Reply:
don't worry about the spacing here... this field is going away - covered in https://issues.folio.org/browse/MODINVOICE-49